docs/get-started/cli-commands.mdx

```diff
@@ -494,6 +494,26 @@ Set block production time in seconds. Default: `2`
 
 ---
 
+<h4><i>access-control-allow-origins</i></h4>
+
+<Tabs>
+  <TabItem value="syntax" label="Syntax" default>
+
+    server [--access-control-allow-origins ACCESS_CONTROL_ALLOW_ORIGINS]
+
+  </TabItem>
+  <TabItem value="example" label="Example">
+
+    server --access-control-allow-origins "*"
```

Comment:
you can specify '*'? what happened if you specify '*'?

Reply:
@PtitLuca from what I can see here: https://github.com/0xPolygon/polygon-edge/pull/368/files we don't support wildcards?

Reply:
@lazartravica I think so